ingeniamotion/configuration.py

```diff
@@ -712,3 +736,145 @@ def restore_tcp_ip_parameters(self, servo: str = DEFAULT_SERVO) -> None:
         """
         drive = self.mc._get_drive(servo)
         drive.restore_tcp_ip_parameters()
+
+    def get_drive_info_coco_moco(
+        self, alias: str
+    ) -> tuple[list[Optional[int]], list[Optional[int]], list[Optional[str]], list[Optional[int]]]:
+        """Get info from COCO and MOCO registers.
```

Comment:
I don't like the word "info" in the function name, but I don't have a better idea. But the function description should be clear, it should be more explicit.

Reply:
Done